src/SWP/Bundle/CoreBundle/Consumer/AnalyticsEventConsumer.php

```diff
@@ -88,11 +88,15 @@ public function execute(AMQPMessage $message)
         $this->setTenant($request);
 
         if ($request->query->has('articleId')) {
-            $this->handleArticlePageviews($request);
+            $this->handleArticlePageViews($request);
+
+            echo 'Pageview for article '.$request->query->get('articleId')." was processed \n";
```

Comment:
why echo here? maybe add logger?

Reply:
we echo i all current sonsumers for now. It's used for now only in console command output test.